src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -93,6 +93,15 @@ class SolventExtractionData(UnitModelBlockData):
         ConfigValue(domain=int, description="Number of finite elements to use"),
     )
 
+    CONFIG.declare(
+        "aqueous_to_organic",
+        ConfigValue(
+            default=True,
+            domain=Bool,
+            description="Direction of the transfer between two phases",
```

Comment:
I actually meant the material transfer is happening from which phase to which phase. So can I write it as whether mass transfer is happening from aqueous phase (True) or organic phase (False) ?

Reply:
If this is meant to be setting the direction of transfer, then a `Bool` is not a suitable way to set this. You would instead want an `Enum`, but finding a concise way to name this would be hard.

Also, I think that is a bit of a misconception really - mass transfer in IDAES is bi-directional although a sign convention indicates which direction it is flowing. However, in this case the issue is less about which way it flows (which in many ways an internal thing) and more about how you are defining the mass transfer term mathematically.

Reply:
Yes it is actually about the mass transfer term being defined on the basis of which phase

Reply:
Andrew's comment on this has not been addressed, but I'm not sure that this alone should hold up the PR